build.gradle

```diff
@@ -15,20 +15,24 @@
  */
 
 buildscript {
+    ext.kotlin_version = '1.2.21'
+
     repositories {
+        google()
         jcenter()
         mavenCentral()
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:2.3.3"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.0"
+        classpath 'com.android.tools.build:gradle:3.2.0-alpha05'
```

Comment:
I'm still using Android Studio 2.3.3. I'm so outdated. This'll force me to explore 3.x. Thanks!